pool/src/lib.rs

```diff
@@ -683,6 +708,46 @@ impl LendingPool {
 
         Ok(())
     }
+
+    fn require_no_pause(env: &Env) -> Result<(), Error> {
+        if paused(env) {
+            return Err(Error::Paused);
+        }
+
+        Ok(())
+    }
+
+    fn is_good_position(
+        env: &Env,
+        who: Address,
+        mb_account_data: Option<AccountData>,
+        value: bool,
+    ) -> Result<bool, Error> {
+        // TODO
+        let _account_data = if let Some(account_data) = mb_account_data {
+            account_data
+        } else {
+            let user_config = read_user_config(env, who.clone())?;
+            let reserves = read_reserves(env);
+            Self::calc_account_data(env, who, &user_config, &reserves)?
+        };
+
+        Ok(value)
+    }
+
+    fn require_good_position(
+        env: &Env,
+        who: Address,
+        mb_account_data: Option<AccountData>,
+        value: bool,
```

Comment:
I't not clear what is this value. Could you provide more details?

Reply:
For now it's just value that method should return because we doesn't have clear vision, what is the criterion for liquidation. Will be removed later